package.json

```diff
@@ -30,70 +31,60 @@
   },
   "license": "MPL-2.0",
   "dependencies": {
-    "codemirror": "5.3.0",
+    "codemirror": "^5.65.0",
     "d3": "^3.1.5",
-    "es6-promise": "2.3.0",
-    "fetch": "1.0.0",
-    "flux": "~2.0.2",
-    "fontawesome": "^4.3.0",
     "jquery": "1.11.3",
     "jquery-ui": "1.12.1",
     "nunjucks": "^3.2.3",
-    "react": "0.13.3",
-    "underscore": "^1.13.1"
+    "underscore": "^1.13.2"
   },
   "devDependencies": {
-    "@babel/cli": "^7.12.16",
-    "@babel/core": "^7.12.16",
-    "@babel/preset-env": "^7.12.16",
-    "@babel/preset-react": "^7.12.13",
-    "@babel/register": "^7.12.13",
+    "@babel/core": "^7.16.7",
+    "@babel/plugin-transform-react-jsx": "^7.16.7",
+    "@babel/preset-env": "^7.16.7",
     "@mozilla-protocol/core": "10.0.1",
-    "autoprefixer": "^9.8.6",
+    "autoprefixer": "^10.4.1",
     "babel-loader": "^8.2.2",
-    "babel-plugin-module-resolver": "^4.1.0",
-    "babelify": "5.0.4",
-    "browser-sync": "^2.26.7",
-    "chai": "3.2.0",
+    "browser-sync": "^2.27.7",
+    "chai": "^4.3.4",
```

Comment:
Among other things (which don't break the tests), dropped support for old node versions: https://github.com/chaijs/chai/releases/tag/4.0.0